Cargo.toml

```diff
@@ -40,7 +40,7 @@ dpdk-sys = { path = "./dpdk-sys" }
 dpdk-sysroot-helper = { path = "./dpdk-sysroot-helper" }
 dplane-rpc = { git = "https://github.com/githedgehog/dplane-rpc.git", version = "1.0.0" }
 errno = { path = "./errno", package = "dataplane-errno" }
-gateway_config = { git = "https://github.com/githedgehog/gateway-proto", version = "0.1.0" }
+gateway_config = { git = "https://github.com/githedgehog/gateway-proto", tag = "v0.2.0", version = "0.1.0" }
```

Comment:
@Fredi-raspall Can you look into this, please? This will conflict with your changes.